src/line/line_data.js

```diff
@@ -178,12 +178,18 @@ function buildToken(builder, text, style, startStyle, endStyle, title, css) {
     }
   }
   builder.trailingSpace = displayText.charCodeAt(text.length - 1) == 32
-  if (style || startStyle || endStyle || mustWrap || css) {
+  if (style || startStyle || endStyle || mustWrap || css || isolate) {
     let fullStyle = style || ""
     if (startStyle) fullStyle += startStyle
     if (endStyle) fullStyle += endStyle
     let token = elt("span", [content], fullStyle, css)
     if (title) token.title = title
+    if (isolate) {
```

Comment:
Will this still work when there are multiple tokens in the range? It seems like it'd create a `<bidi>` tag for each token separately, which I believe won't get the result we want.

Reply:
You're right, I'll try to find a solution to that. Is there an option to prevent certain markers from overlapping each other?

Reply:
I added a function to check that isolate ranges do not overlap, in the same style as conflictingCollapsedRange()